Content.Shared/_ES/Auditions/CharacterComponent.cs

```diff
@@ -0,0 +1,30 @@
+using Content.Shared.Humanoid;
+using Robust.Shared.Enums;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._ES.Auditions;
+
+/// <summary>
+/// This is used for marking the character of components.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class CharacterComponent : Component
```

Comment:
needs proper networking via AutoNetworkedComponent and AutoNetworkedField on all the datafields. Plus, you need to call Dirty when these fields are modified.

not having this on the client is gonna make the UI a PITA when i get to it.